backend/src/controllers/user.ts

```diff
@@ -64,23 +37,7 @@ export const updatePassword = async (
   next: NextFunction
 ) => {
   try {
-    const email = req.params.userEmail;
-    const user = await User.update(
-      { password: req.body.password },
-      { where: { email: req.params.userEmail } }
-    );
-
-    if (user) {
-      const userValue = user.at(0);
-      if (userValue === 0) {
-        res.status(403).send({
-          message: "User " + email + " does not exists!",
-        });
-      } else {
-        res.status(200);
-        res.send("Your password is updated!");
-      }
-    }
+    handleUpdatePassword(req, res);
```

Comment:
>     handleUpdatePassword(req, res);

No need to pass req & res to the handler.
Pass only the necessary arguments to the handler